src/test/ui/dst/dst-object-from-unsized-type.stderr

```diff
@@ -6,7 +6,7 @@ LL | fn test1<T: ?Sized + Foo>(t: &T) {
 LL |     let u: &dyn Foo = t;
    |                       ^ doesn't have a size known at compile-time
    |
-   = help: the trait `std::marker::Sized` is not implemented for `T`
+   = help: type parameters have an implicit `Sized` requirement by default
```

Comment:
Huh, these help messages seem to be kind of confusing to me. That is, just looking at the error message, I can't really tell what it's talking about here.